backend/tests/api/test_users.py

```diff
@@ -0,0 +1,132 @@
+from app.auth.config import API_SECRET_KEY
+from app.users.routers import router
+from fastapi.testclient import TestClient
+
+client = TestClient(router)
+
+
+def test_create_user(client: TestClient) -> None:
```

Comment:
The biggest issue is that the tests dont clean up after themselves. So they may start interfering with other tests as content will remain the db.

Suggest using a fixture that just yields and the cleans the db by running a DELETE SQL statement